imports/main.html

```diff
@@ -5,15 +5,22 @@
 <body>
   <div class="top-bar">
     <div class="top-bar-left">
-      <span class="top-bar-title"><b>Leaderboard</b>
-        <button class="button" id="addPlayerModalButton" data-open="AddPlayerModal">Add Player</button>
-      </span>
+      <ul class="dropdown menu" data-dropdown-menu>
+        <li class="menu-text">Leaderboard</li>
+        <li><a href="#" class="button small" id="addPlayerModalButton" data-open="AddPlayerModal">Add Player</a></li>
 
+      </ul>
     </div>
-    <div class="top-bar-right">{{>loginButtons}}</div>
```

Comment:
took that away from here as it often pops open into the off screen area. now placed in middle of content if not logged in... could be changed